src/aosm/azext_aosm/custom.py

```diff
@@ -7,32 +7,42 @@
 from pathlib import Path
 from azext_aosm.cli_handlers.onboarding_cnf_handler import OnboardingCNFCLIHandler
 from azext_aosm.cli_handlers.onboarding_vnf_handler import OnboardingVNFCLIHandler
+from azext_aosm.cli_handlers.onboarding_core_vnf_handler import OnboardingCoreVNFCLIHandler
+from azext_aosm.cli_handlers.onboarding_nexus_vnf_handler import OnboardingNexusVNFCLIHandler
 from azext_aosm.cli_handlers.onboarding_nsd_handler import OnboardingNSDCLIHandler
 from azext_aosm.common.command_context import CommandContext
 from azext_aosm.common.constants import ALL_PARAMETERS_FILE_NAME, CNF, VNF
 from azure.cli.core.commands import AzCliCommand
 from azure.cli.core.azclierror import UnrecognizedArgumentError
 
 
-def onboard_nfd_generate_config(definition_type: str, output_file: str | None):
+def onboard_nfd_generate_config(definition_type: str, output_file: str | None, nexus: bool = False):
```

Comment:
As mentioned in `_params.py` comment, I think treating this as another `definition-type` would be better, in which case this file would need updating too.